docs/guides/headless-cms/build-static-website/next-13.md

```diff
@@ -49,9 +49,9 @@ To share a single instance of the Directus JavaScript SDK between multiple pages
 file that can be imported later. Create a new directory called `lib` and a new file called `directus.js` inside of it.
 
 ```js
-import { Directus } from '@directus/sdk';
+import { createDirectus } from '@directus/sdk';
 
```

Comment:
You're missing the importing of the rest function

Reply:
Good catch. I just updated the docs with the imports

Reply:
But, also, is it enough to import them here? Or do you need to import them where they are used. Please test just to make sure